addons/mail/static/src/utils/common/format.js

```diff
@@ -276,10 +295,17 @@ export function getNonEditableMentions(body) {
     // for mentioned partner
     for (const mention of doc.body.querySelectorAll(".o_mail_redirect")) {
         mention.setAttribute("contenteditable", false);
+        mention.setAttribute("draggable", false);
```

Comment:
Shouldn't that be a fix on its own PR too? Why do we need that? Is it covered by a test?